source/dub/commandline.d

```diff
@@ -538,6 +555,8 @@ struct CommonOptions {
 	bool help, annotate, bare;
 	string[] registry_urls;
 	string root_path;
+	enum colors { automatic, on, off } // Style violation in support of invalid option error formatting.
```

Comment:
the style violation for the `enum colors` should not affect how invalid options are formatted, right? (e.g. renaming colors to Colors should not have any effect)

Reply:
It does have effect. If a user supplies a value that is not part of the enum, a message is generated in which the name of the enum occurs (here `color`). Because the option is `--color=`,  it makes sense to use the same name in the error message.